online_store/gt_sport_v2.js

```diff
@@ -0,0 +1,182 @@
+    },
+    availability: 10,
+  },
+  {
+    imgSrc: "volleyball.jpg",
+    name: "Volleyball ball",
+    brand: "Mikasa",
+    level: "beginner",
+    price: 26,
+    seller: "Mikasa",
+    other: {
+      helmetNeeded: false,
+      longevity: "short-term",
+      teamSpirit: true,
+    },
+    availability: 3,
+  },
+  {
+    imgSrc: "xbadmintonset.jpg",
+    name: "Badminton set",
+    brand: "Talbot Torro",
+    level: "intermediate",
+    price: 28,
+    seller: "Talbot Torro",
+    other: {
+      helmetNeeded: false,
+      longevity: "mid-term",
+      teamSpirit: true,
+    },
+    availability: 5,
+  },
+  {
+    imgSrc: "skigoggles.jpg",
+    name: "Ski Goggles",
+    brand: "Lemego",
+    level: "intermediate",
+    price: 28,
+    seller: "Lemego",
+    other: {
+      helmetNeeded: true,
+      longevity: "mid-term",
+      teamSpirit: true,
+    },
+    availability: 7,
+  },
+];
+const productDescription = document.getElementById("productDescription");
+let shoppingCart = [];
+let productEl = [];
+const sportProductsEl = document.getElementById("sportProducts");
+const addToShoppingCartEl = document.getElementById("addToShoppingCart");
+let totalPriceEl = document.getElementById("totalPrice");
+let shoppingCartEl = document.getElementById("shoppingCart");
+let calTPbtn = document.getElementById("calTP");
+let addToShoppingCartBtn = document.getElementById("addToShoppingCartBtn");
+let descriptions = [];
+let intoSCbtn = document.getElementById("intoSC");
+
+function productDescriptionDisplay() {
+  for (let i = 0; i < sportProducts.length; i++) {
+    descriptions = document.createElement("div");
+    const btnEl = document.createElement("button");
+    const product = sportProducts[i];
+
+    if (sportProducts[i].availability < 1) {
+      descriptions.innerHTML = `
+      <p> <b>${sportProducts[i].name}</b></p>
+      <p> <img src='${sportProducts[i].imgSrc}' style="width:50px"><p>
+      <p> <b>Brand:</b> ${sportProducts[i].brand}</p>
+      <p> <b>Price:</b> ${sportProducts[i].price} Eur</p>
+      <p> Sold by a <b>gT partner</b> called ${sportProducts[i].seller}.</p>
+      <p> This ${sportProducts[i].name} is suitable for teenagers on ${sportProducts[i].level} level.</p>
+      <p> This product is currently out of stock / not available. </p>
+      `;
+    }
+    else if (sportProducts[i].availability === 1) {
+      descriptions.innerHTML = `
+      <p> <b>${sportProducts[i].name}</b></p>
+      <p> <img src='${sportProducts[i].imgSrc}' style="width:50px"><p>
+      <p> <b>Brand:</b> ${sportProducts[i].brand}</p>
+      <p> <b>Price:</b> ${sportProducts[i].price} Eur</p>
+      <p> Sold by a <b>gT partner</b> called ${sportProducts[i].seller}.</p>
+      <p> This ${sportProducts[i].name} is suitable for teenagers on ${sportProducts[i].level} level.</p>
+      <p> Last chance! Only 1 more left in stock! </p>
+      `;
+    }
+    else if (sportProducts[i].availability > 1 && sportProducts[i].availability <= 3) {
```

Comment:
each of these if blocks has a ton of duplication. perhaps you could use a function or different structuring to repeat yourself less.

Reply:
Thanks